pos_printer_network/static/src/js/network_printer.js

```diff
@@ -130,7 +130,17 @@ odoo.define('pos_restaurant.network_printer', function (require) {
                 self.network_printers.push({'ip': item.config.proxy_ip, 'status': 'offline', 'name': item.config.name});
             });
 
-            if (!this.pos.config.usb_printer_active) {
+            var exist_usb_printer = this.pos.printers.find(function(printer){
+                return !printer.config.network_printer;
+            });
+
+            if (exist_usb_printer) {
+                this.pos.usb_printer_active = true;
+            } else {
+                this.pos.usb_printer_active = false;
+            }
+
+            if (!this.pos.usb_printer_active) {
```

Comment:
this.pos.usb_printer_active = this.pos.printers.find(function(printer){
                return !printer.config.network_printer;
            });

current POS